_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -88,8 +88,96 @@ Option | Required | Type | Description
+* `partition_key` - The partition key of the DynamoDB item
+* `sort_key` - The sort key of the DynamoDB item. This will be null if the table does not have a sort key
+* `dynamodb_timestamp` - The timestamp of the DynamoDB item. This will be the export time for export items, and the DDB stream Event time for stream items. This timestamp is used by sinks to emit an `EndtoEndLatency` metric for DDB stream Events, which tracks the latency between a change in the DDB table, and that change getting applied to the sink.
+* `document_version` - Based off the `dynamodb_timestamp`, and modified to break ties between stream items that are received in the same second. Recommend for use with the `opensearch` sink `document_version` setting.
+* `opensearch_action` - A default value for mapping DDB Event actions to OpenSearch actions. This action will be `index` for export items, and INSERT or MODIFY stream events. This action will be `delete` for REMOVE stream events.
+* `dynamodb_event_name` - The exact event type for the item. Will be `null` for export items, and either `INSERT`, `MODIFY`, or `REMOVE` for stream events.
+* `table_name` - The DynamoDB table name that an event came from.
+
+
+## Permissions
+
+The following shows the minimum required permissions for running DDB as a source
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "allowRunExportJob",
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:DescribeTable",
+                "dynamodb:DescribeContinuousBackups",
+                "dynamodb:ExportTableToPointInTime"
+            ],
+            "Resource": [
+                "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table"
+            ]
+        },
+        {
+            "Sid": "allowCheckExportjob",
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:DescribeExport"
+            ],
+            "Resource": [
+                "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table/export/*"
+            ]
+        },
+        {
+            "Sid": "allowReadFromStream",
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:DescribeStream",
+                "dynamodb:GetRecords",
+                "dynamodb:GetShardIterator"
+            ],
+            "Resource": [
+                "arn:aws:dynamodb:us-east-1:{account-id}:table/my-table/stream/*"
+            ]
+        },
+        {
+            "Sid": "allowReadAndWriteToS3ForExport",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:AbortMultipartUpload",
+                "s3:PutObject",
+                "s3:PutObjectAcl"
+            ],
+            "Resource": [
+                "arn:aws:s3:::my-bucket/*"
+            ]
+        }
+    ]
+}
+```
+
```

Comment:
It might be useful to add a paragraph here that tells users they can remove Sid `allowReadFromStream` when doing an export only. And also remove the correct Sid's (`allowRunExportJob`, `allowCheckExportjob`, `allowReadAndWriteToS3ForExport`)  when streaming only.